frontend/src/features/counter/Counter.module.css

```diff
@@ -0,0 +1,74 @@
+.row {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
```

Comment:
This might be the default setting but wanted to ask. Are you wanting to use css modules or do you want to use a preprocessor like SASS? There is merits to both and its ultimately up to you guys but thought to ask!

Reply:
Yeah this is the default that came out of the cra template, I've never actually used the modules thing. I think I might leave it in and deffer a decision to whenever we get to start styling some components. Plus not sure what the others would be comfortable with using most.